vm/base-context.cpp

```diff
@@ -236,3 +236,14 @@ BasePluginContext::BlamePluginError(SourcePawn::IPluginFunction *pf, const char
   va_end(ap);
   return 0;
 }
+
+IFrameIterator *BasePluginContext::GetFrameIterator()
```

Comment:
Local style is:

```
IFrameIterator *
BasePluginContext::GetFrameIterator()
```

(Return type/modifiers on one line, class/method name on next)